aea/skills/base.py

```diff
@@ -437,9 +437,9 @@ def act_wrapper(self) -> None:
             raise
         except Exception as e:  # pylint: disable=broad-except
             e_str = parse_exception(e)
-            raise AEAActException(
-                f"An error occurred during act of behaviour {self.context.skill_id}/{type(self).__name__}:\n{e_str}"
-            )
+            err_msg = f"An error occurred during act of behaviour {self.context.skill_id}/{type(self).__name__}:\n{e_str}"
+            self.context.logger.exception(err_msg)
+            raise AEAActException(err_msg)
```

Comment:
why is this needed? AEAActException bubbles up and then depending on logging policy you can achieve the same (e.g. by setting `just_log`)